include/natalie/kernel_module.hpp

```diff
@@ -6,6 +6,8 @@
 
 #include "natalie/forward.hpp"
 #include "natalie/value.hpp"
+#include "natalie/integer_value.hpp"
+#include "natalie/nil_value.hpp"
```

Comment:
These two were missing (probably included in the other headers/source files), but I added them here anyway